apps/dev-recruiters/src/components/modules/FilterPage/RolesFilterComponent/RolesFilterList/CollapsibleContainerFilter/index.tsx

```diff
@@ -43,21 +50,33 @@ const CollapsibleContainerFilter: React.FC<CollapsibleContainerProps> = ({
             <RolesButtons
               textRole="Product Lead"
               onClick={() =>
-                handleRoleClick('ProductLead', openPositions['ProductLead'])
+                handleRoleClick(
+                  'ProductLead',
+                  openPositions['ProductLead'],

```

Comment:
Does the key need to match the enum value in the backend https://github.com/dev-launchers/strapiv4/blob/main/src/api/opportunity/content-types/opportunity/schema.json#L73?

Reply:
Oh the actual mapping to the backend values happen in apps/dev-recruiters/src/components/modules/FilterPage/RolesFilterComponent/RolesFilterList/UseOpenPositions/opportunityUtils.tsx?

Reply:
Yes exactly.